src/router/index.ts

```diff
@@ -9,15 +10,8 @@ const router = createRouter({
       name: 'tiny-mce',
       component: TinyMCE
     },
-    // {
-    //   path: '/about',
```

Comment:
The code looks generally good, but there are a few minor points to consider. The commented-out route for '/about' has been removed, which is good for cleanliness. However, if this route might be needed in the future, consider adding a TODO comment. The added comment about additional routes is helpful for maintainability.

``` 
// TODO: Add more routes as needed
// Example:
// {
//   path: '/about',
//   name: 'about',
//   component: () => import('../views/AboutView.vue')
// }
```